### PR TITLE
Update PSM link, CUs, and replaced changelog with chainlog.

### PR DESCRIPTION
**Pull request by jeannettemcd**
- Updated link to PSM
- Updated CUs, and alphabetized
- Replaced changelog with chainlog link

### CODE REVIEW

File: src/commands/coreunitCommand.js [hunk 1]
```diff
@@ -11,25 +11,26 @@ module.exports = {
...
+			['IS-001: Immunefi Security', 'Immunefi Security'],
+			['MDS-001: MakerDAO Shop', 'MakerDAO Shop'],
 			['ORA-001: Oracles', 'Oracles'],
 			['PE-001: Protocol Engineering', 'Protocol Engineering'],
-			['RWF-001: Real World Finance', 'Real World Finance'],
 			['RISK-001: Risk', 'Risk'],
+			['RWF-001: Real World Finance', 'Real World Finance'],
+			['SAS-001: Sidestream Auction Services', 'Sidestream Auction Services'],
 			['SES-001: Sustainable Ecosystem Scaling', 'Sustainable Ecosystem Scaling'],
+			['SF-001:  Strategic Finance', 'Strategic Finance'],
```
Comment (sanbotto) [inline src/commands/coreunitCommand.js:30]:
```suggestion
			['SF-001: Strategic Finance', 'Strategic Finance'],
```

File: src/commands/coreunitCommand.js [hunk 2]
```diff
@@ -11,25 +11,26 @@ module.exports = {
 		.setDescription('Core Unit name')
 		.addChoices([
 			['CES-001: Collateral Engineering', 'Collateral Engineering Services'],
+			['COM-001: Governance Communications', 'Governance Communications'],
 			['DAIF-001: Dai Foundation', 'Dai Foundation'],
+			['DECO-001:  Deco Fixed Rate', 'Deco Fixed Rate'],
```
Comment (sanbotto) [inline src/commands/coreunitCommand.js:16]:
```suggestion
			['DECO-001: Deco Fixed Rate', 'Deco Fixed Rate'],
```

**Review by sanbotto — APPROVED** [review]
Looks good but I pointed out two double spaces in some names just in case...

File: src/commands/changelogCommand.js [hunk 3]
```diff
@@ -1,24 +1,11 @@
...
-        interaction.reply({ content: 'Getting the changelog...', ephemeral: true })
-        await wait(1000);
-        
-        const data = await fetchChangelog(key)
-        interaction.editReply({ content: '```css\n'+ data +  '```', ephemeral: true })
-
-        
-
-        console.log('done')
+        return interaction.reply({ embeds: [{"title": 'MakerDAO Changelog', "description": 'https://chainlog.makerdao.com/', "color": 15258703}], ephemeral: true })
```
Comment (zdumitru) [inline src/commands/changelogCommand.js:9]: Shouldn't the title be "MakerDAO Chainlog"?

**Pull request merged**